pypeit/core/wavecal/waveio.py

```diff
@@ -107,7 +108,9 @@ def load_reid_arxiv(arxiv_file):
     """
     # This function allows users to specify their own `reid_arxiv`, in
     #   particular, the output from `pypeit_identify`.
-    calibfile, arxiv_fmt = data.get_reid_arxiv_filepath(arxiv_file)
+    # NOTE: I use to_pkg='symlink' here so that the extension of the file is correct.
+    calibfile, arxiv_fmt = dataPaths.reid_arxiv.get_file_path(arxiv_file, return_format=True,
+                                                              to_pkg='symlink')
```

Comment:
Good question.  The tests seem to run okay in the cloud, but I agree this may be a problem for system-level installations of pypeit.  I think I've now limited the use of this to the test functions, meaning this should not affect a general user.